schema/partner_show.js

```diff
@@ -82,6 +89,18 @@ const PartnerShowType = new GraphQLObjectType({
     status: {
       type: GraphQLString,
     },
+    status_update: {
+      type: GraphQLString,
+      description: 'A formatted update on upcoming status changes',
+      args: {
+        max_days: {
+          type: GraphQLInt,
+          description: 'Before this many days no update will be generated',
```

Comment:
"Before this many days no update will be generated" doesn't make too much sense, perhaps "the maximum amount of days to show a relative day string, instead of the full time range"?